### PR TITLE
change in request getting data from modal to getting from repository

### PR DESCRIPTION
**Pull request by Roksoliana**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/browse/LVPHPIII-XXX)
* [JIRA subtask](https://ssu-jira.softserveinc.com/browse/LVPHPIII-XXX)

## Code reviewers

- [ ] @ecconomix
- [ ] @kovalc0mrade
- [ ] @NFalinska
- [ ] @NikitaKompanetc
- [ ] @nykolaiko
- [ ] @Roksoliana
- [ ] @topvova

### Second Level Review

- [ ] @masJIan
- [ ] @lhalam

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST

- [ ]  СI passed
- [ ]  Сode coverage >=90%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: app/Http/Controllers/v1/RequestController.php [hunk 1]
```diff
@@ -14,11 +14,14 @@ class RequestController extends Controller
...
     /**
      * @param Request $request
+     * @param RequestRepository $requestRepository
      * @return \Illuminate\Http\JsonResponse
      */
-    public function index(Request $request)
+    public function index(Request $request, RequestRepository $requestRepository)
     {
-        return response()->json(['request'=>$request->all()]);
+//        $user = 5;
```
Comment (masJIan) [inline app/Http/Controllers/v1/RequestController.php]: ?

File: app/Repositories/RequestRepository.php [hunk 2]
```diff
@@ -17,4 +18,8 @@ protected function model()
     {
         return Request::class;
     }
+    public function userRequest($user)
```
Comment (masJIan) [inline app/Repositories/RequestRepository.php]: Annotations

File: app/Http/Controllers/v1/PassportController.php [hunk 3]
```diff
@@ -33,9 +31,9 @@ public function show(PassportRepository $passportRepository, $id)
 
         if ($passportId) {
             return response()->json([
-                'passport_name'        =>$passportId->name,
-                'passport_icon'        =>$passportId->icon,
-                'passport_description' =>$passportId->description,
+                'name'        =>$passportId->name,
```
Comment (kovalc0mrade) [inline app/Http/Controllers/v1/PassportController.php]: витягуєш з Ід імя? перейменуй в $passport

File: app/Http/Controllers/v1/RequestController.php [hunk 4]
```diff
@@ -30,4 +30,15 @@ public function show(RequestRepository $requestRepository, $id)
     {
         return response()->json(['Statuses' => $requestRepository->getDetailsInformation($id)]);
```
Comment (kovalc0mrade) [inline app/Http/Controllers/v1/RequestController.php]: форматування

File: app/Http/Controllers/v1/RequestController.php [hunk 5]
```diff
@@ -30,4 +30,15 @@ public function show(RequestRepository $requestRepository, $id)
...
     }
+
+    /**
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function usersRequests(Request $request)
+    {
+        return response()->json([
+            'request'   =>  $request->user()->requests()->get()
```
Comment (kovalc0mrade) [inline app/Http/Controllers/v1/RequestController.php]: ключ request  а ти пишеш в нього масив реквестів. перейменуй в requests
Для чого ту метод get()?

File: app/Repositories/RequestRepository.php [hunk 6]
```diff
@@ -19,8 +20,7 @@ protected function model()
     }
     public function getDetailsInformation($id)
```
Comment (kovalc0mrade) [inline app/Repositories/RequestRepository.php]: code annotation
метод getDetailsInformation
повертаєш defaultStatuses
а працюєш з RequestRepository
WTF??

**Pull request merged**